mainWindow.py

```diff
@@ -86,17 +83,15 @@ def openFile(self, tableObj):
         if not utilities.file_writable(fileB):
             print(fileB + ": Write permission denied.")
 
-        # prompt = QMessageBox.about(self, "Error", "Error Message")
-
-        #fileA = fileOpener.fileAName
-        #fileB = fileOpener.fileBName
         fileA = fileOpener.fileAName
         fileB = fileOpener.fileBName
 
         result = self.fIO.diffFiles(fileA, fileB)
 
         if result == pmEnums.RESULT.GOOD:
             result = self.fIO.getChangeSets(self.fIO.changesA, self.fIO.changesB)
+        elif result == pmEnums.RESULT.BADFILE:
+            QMessageBox.about(self, "Error", "Invalid file type")
```

Comment:
Maybe include the file name here so the user knows which file caused the error. Nicely done.